Lib/fontTools/varLib/errors.py

```diff
@@ -61,6 +78,9 @@ def __str__(self):
             details = f"\n\nThe problem is likely to be in {offender}:\n"
         if cause["reason"] == VarLibMergeFailure.FoundANone:
             details = details + f"{stack[0]}=={cause['got']}\n"
+        elif cause["reason"] == VarLibMergeFailure.ShouldBeConstant:
```

Comment:
is the `ShouldBeConstant` case only ever used for incompatible features between masters that is being singled out below as "common case"? Otherwise it'd be pretty confusing if the non-constant value is elsewhere but the error message says "Incompatible features between masters" and the two lists are in fact identical